### ISSUE TITLE
C# error messages are sentences. Most of them end with a period. Some of them are missing the period. We should be consistent.

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#622]

**Comment by MattWhilden** [comment]
So... Remove the periods or add the periods?

**Comment by gafter** [comment]
@MattWhilden Yes. Personally I would add them.

**Comment by kirisky** [comment]
Hi! 

Does this only involve adding periods to the CSharp part? Do the Core and Visual Basic sections need changes as well? Or are there other parts that need to be modified too?

**Comment by cston** [comment]
I don't think we should fix this. The potential cost (for instance, for localization, and potential merge conflicts) is likely to far exceed the value.
